ProjectSettings/ProjectSettings.asset

```diff
@@ -99,7 +99,7 @@ PlayerSettings:
   xboxEnableFitness: 0
   visibleInBackground: 1
   allowFullscreenSwitch: 1
-  fullscreenMode: 1
+  fullscreenMode: 0
```

Comment:
Is this just the editor mode or is this the default for the game and why is it changed?

Reply:
Default, and that is because the background looks the best it can be in exclusive fullscreen (was maximized window). You can see what I mean when you exit out of fullscreen mode in a build that contains these changes.

Reply:
Alright then this is fine.